acceptance/acceptance_test.go

```diff
@@ -413,11 +411,20 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	testdiff.PrepareReplacementsUser(t, &repls, user)
 	testdiff.PrepareReplacementsWorkspaceClient(t, &repls, workspaceClient)
 
-	// Must be added PrepareReplacementsUser, otherwise conflicts with [USERNAME]
-	testdiff.PrepareReplacementsUUID(t, &repls)
+	// Must be added after PrepareReplacementsUser, otherwise conflicts with [USERNAME]
+	// for service principal users.
+	if isTruePtr(config.DisableComparableUuidReplacement) {
+		// bin/diff.py does not support stateful replacements. We provide a flag
```

Comment:
We can also consider modifing bin/diff.py to 
- replace "[[UUID-N]]" with "[[UUID]]"
- replace uuid regex with "[[UUID]]"
before comparison.

Then it won't be sensitive to N in "[[UUID-N]]" and we can don't need an option to disable the feature because of this.